landing-pages/site/content/en/blog/airflow-2.7.0/index.md

```diff
@@ -0,0 +1,87 @@
+
+There is a new top level page in Airflow, the Cluster Activity page. This gives an overview of the cluster, including component health, dag and task state counts, and more!
+
+![New cluster activity page](cluster_activity.png)
+
+## Graph and gantt views moved into the Grid view UI
+
+The graph and gantt views have been rewritten and moved into the now familiar grid view. This makes it easier to jump between task details, logs, graph, and gantt views without losing your place in a complicated DAG.
+
+![Graph in grid view](graph_in_grid.png)
+
+## Enable deferrable mode for all deferable tasks with 1 config setting
+
+Airflow 2.7.0 comes with a new config option, `default_deferrable`, which allows admins to enable deferrable mode for all deferrable tasks without requiring any DAG modifications. Simply set it in your config and enjoy async tasks!
+
+## OpenLineage built-in integration
+
+[OpenLineage](https://openlineage.io/) provides a spec standardizing operational lineage collection and distribution across the data ecosystem that projects – open source or proprietary – implement.
+
+With 2.7.0, OpenLineage changes from a plugin implementation maintained in the OpenLineage project to a built-in feature of Airflow. As a plugin, OpenLineage depended on Airflow and operators’ internals, making it brittle. Built-in OpenLineage support in Airflow makes publishing operational lineage through the OpenLineage ecosystem easier and more reliable. It has been implemented by moving the [openlineage-airflow](https://github.com/OpenLineage/OpenLineage/tree/main/integration/airflow) package from the OpenLineage project to an `apache-airflow-providers-openlineage` provider in the base Airflow Docker image, where it can be easily enabled by configuration. Also, lineage extraction logic that was included in [Extractors](https://github.com/OpenLineage/OpenLineage/tree/main/integration/airflow/openlineage/airflow/extractors) in that package has been moved into each corresponding provider package along with unit tests, eliminating the need for Extractors in most cases. For this purpose, a new optional API for Operators (`get_openlineage_facets_on_{start(), complete(ti), failure(ti)}`, documented [here](https://openlineage.io/docs/integrations/airflow/default-extractors)) can be used. Having the extraction logic in each provider ensures the stability of the lineage contract in each operator and makes adding lineage coverage to custom operators easier.
+
+## Some executors moved into providers
+
+Some of the executors that were shipped in core Airflow have moved into their respective providers for Airflow 2.7.0.
+
+The following providers have been moved and require certain minimum providers versions:
+
+* In order to use Celery executors, install the celery provider version 3.3.0+
+* In order to use the Kubernetes executor, install the kubernetes provider version 7.4.0+
+* In order to use the Dask executor, install any version of the daskexecutor provider
```

Comment:
I don't find links to version in pypi that helpful in general. Maybe link to the provider changelog instead?

Reply:
The PyPI page is actually pretty useful and usable (especially after some recent changes)

Including quite easily findable "Changelog" link (also repeated at the bottom of rather short information page).

<img width="1512" alt="Screenshot 2023-08-18 at 17 05 41" src="https://github.com/apache/airflow-site/assets/595491/df1e3678-d42c-4f23-9ae1-c12936f67519">

Reply:
Check it out @jedcunningham -> You might have not realized we shortened it and separated Changelog to separate page for providers and added links to it instead of having it all there (It's been far too long and messy before).